doc/TESTS.rst.txt

```diff
@@ -327,7 +327,7 @@ To skip a test, simply use ``skipif``::
 
   from numpy.testing import dec
 
-  @dec.skipif(SkipMyTest, "Skipping this test because...")
+  @pytest.mark.skipif(SkipMyTest, "Skipping this test because...")
```

Comment:
IIRC, you need the `reason=` keyword with the reason.

Reply:
Thanks! I will double check and update it.